tests/examples/tokens/test_erc721.py

```diff
@@ -186,6 +186,26 @@ def test_safeTransferFrom_by_owner(c, w3, assert_tx_failed, get_logs):
     assert c.balanceOf(operator) == 2
 
 
+def test_safeTransferFrom_overload_function(c, w3, assert_tx_failed, get_logs):
+    someone, operator = w3.eth.accounts[1:3]
+
+    # transfer by owner
+    tx_hash = c.safeTransferFrom(
+        someone, operator, SOMEONE_TOKEN_IDS[0], b"101", transact={"from": someone}
```

Comment:
Can you also create another contract instead of operator that would record the value of data from the callback so we can see it?